ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-ref.yaml

```diff
@@ -66,6 +66,9 @@ ref:
   - name: PRIVATE_LINK
     default: "false"
     documentation: Provides private connectivity between VPCs, AWS services, and your on-premises networks, without exposing your traffic to the public internet.
+  - name: USER_TAGS
+    default: ""
+    documentation: Enable you to tag your clusters, if none provided defaults to "ciprow_cluster:${CLUSTER_NAME}"
```

Comment:
Use the option description here: ` Apply user defined tags to all resources created by ROSA in AWS. Tags are comma separated, for example: 'foo:bar,bar:baz'. The default value is "cluster:${CLUSTER_NAME}".`

Reply:
Update, ptal